docs/user/library/coverage/imageio.rst

```diff
@@ -52,16 +54,83 @@ file formats:
 * RPF TOC
 * SRP (ASRP/USRP) formats
 
-The ImageIO-EXT website includes ready to use ImageIO-Ext binaries including GDAL libraries. See
-"Release Information" for more info about the list of supported formats and available drivers.
+To make use of gdal-bridge functionality:
```

Comment:
Don't think we are calling this a "gdal-bridge" anywhere else, I'd avoid adding new terminology:

```suggestion
To make use of the GDAL based coverage readers:
```